packages/webapp/src/routes/__tests__/credits-purchase/date-of-birth.spec.js

```diff
@@ -85,7 +91,7 @@ describe(url, () => {
       postOptions.payload['dob-year'] = goodYear
       const res = await submitPostRequest(postOptions, 200)
       expect(res.payload).toContain('There is a problem')
-      expect(res.payload).toContain('Date of birth must be a real date')
+      expect(res.payload).toContain('Date of birth must be a number')

```

Comment:
Date of birth must be a number is not the correct error message. old message is the correct one